azure-pipelines.yml

```diff
@@ -57,11 +57,6 @@ steps:
       yarn size:snapshot
     displayName: 'create a size snapshot'
 
-  - task: PublishPipelineArtifact@0
-    inputs:
-      artifactName: 'size-snapshot'
-      targetPath: 'scripts/sizeSnapshot/build/stats.json'
```

Comment:
I didn't use this for months. It takes now up to 1min extra CI time. Since they are only persisted for 30 days anyway and we can safely reproduce them (since we have a lockfile) if need be I removed it.